marklogic-data-hub/src/main/java/com/marklogic/hub/impl/ScaffoldingImpl.java

```diff
@@ -173,6 +174,29 @@ public static String getAbsolutePath(String first, String... more) {
         }
     }
 
+    public Pair<File, String> addStepToFlow(String flowName, String stepName, String stepType) {
```

Comment:
This looks good, but can you move it to FlowManagerImpl? I think the scope of ScaffoldingImpl should be limited to scaffolding out new file in a project, whereas this is a flow-specific operation.